debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/spi/OffsetState.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.postgresql.spi;
+
+import java.time.Instant;
+
+/**
+ * A simple data container that represents the last seen offset
+ * which was written by debezium.
+ */
+public class OffsetState {
```

Comment:
Can you add a new annotation `io.debezium.annotation.Incubating` (in _debezium-core_) for marking incubating APIs/SPIs and put it to all the new types in this SPI package.

Reply:
This one can serve as example: https://github.com/hibernate/hibernate-validator/blob/master/engine/src/main/java/org/hibernate/validator/Incubating.java